application/src/main/java/org/togetherjava/tjbot/features/moderation/TransferQuestionCommand.java

```diff
@@ -89,12 +90,14 @@ public void onMessageContext(MessageContextInteractionEvent event) {
             .setValue(createTitle(originalMessage))
             .build();
 
-        TextInput modalInput =
+        Builder modalInputBuilder =
                 TextInput.create(MODAL_INPUT_ID, "Question", TextInputStyle.PARAGRAPH)
-                    .setValue(originalMessage)
                     .setRequiredRange(INPUT_MIN_LENGTH, INPUT_MAX_LENGTH)
-                    .setPlaceholder("Contents of the question")
-                    .build();
+                    .setPlaceholder("Contents of the question");
+
+        if (!isQuestionTooShort(originalMessage)) {
+            modalInputBuilder.setValue(originalMessage);
+        }
```

Comment:
what if its too long? 😬

Reply:
ofc didn't even think of that. Now it makes sense, we gotta cover the edge cases since discord is relying on us not the user input.